packages/fxa-payments-server/src/components/PaymentErrorView/index.tsx

```diff
@@ -23,6 +21,34 @@ export const PaymentErrorView = ({
   error,
   className = '',
 }: PaymentErrorViewProps) => {
+  const history = useHistory();
+
+  // We want the button label and onClick handler to be different depending
+  // on the type of error
+  const ActionButton = () => {
+    return error?.code === 'returning_paypal_customer_error' ? (
+      <Localized id="payment-error-manage-subscription-button">
+        <button
+          data-testid="manage-subscription-link"
+          className="button"
+          onClick={() => history.push('/subscriptions')}
```

Comment:
I used `history.push` here, though I don't see the `useHistory` hook used elsewhere. Is there a better/preferred way to route to the `'/subscriptions'` relative URL?

Reply:
We could also use the [Link Component](https://reactrouter.com/web/api/Link)

Reply:
Thanks for the suggestion. Hmm... I don't see that used anywhere else either. If I wrap the `<button>` in a `<Link>` to achieve this, the styles break, so if it's all the same to you, I'll just keep the `useHistory` hook.